GitTank/ViewModels/TabWithLogsViewModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace GitTank.ViewModels
+{
+    public class TabWithLogsViewModel : BaseViewModel
+    {
+        public string Header { get; set; }
```

Comment:
It will work as is as we do not assume that header might change. However, I would implement OnPropertyChanged anyway to eliminate potential defect when we add support for renaming repos.